src/super_gradients/training/utils/media/stream.py

```diff
@@ -30,7 +34,11 @@ def __init__(
         self.frame_processing_fn = frame_processing_fn
         self.cap = cv2.VideoCapture(capture)
         if not self.cap.isOpened():
-            raise ValueError("Could not open video capture device")
+            message = "Could not open video capture device. Please check whether you have the webcam connected."
+            if sys.platform == "darwin":
```

Comment:
Actually, I think windows also blocks the camera. maybe not by default, but it might also be blocked

Reply:
https://support.microsoft.com/en-us/windows/manage-app-permissions-for-your-camera-in-windows-87ebc757-1f87-7bbf-84b5-0686afb6ca6b#:~:text=Select%20Start%20%3E%20Settings%20%3E%20Privacy%20%26,able%20to%20access%20the%20camera.

Reply:
Improved in 2697e1f44fb6f8d64197ac147d03d10dd628b423